src/coreclr/clrfeatures.cmake

```diff
@@ -31,3 +31,7 @@ endif(NOT DEFINED FEATURE_AUTO_TRACE)
 if(NOT DEFINED FEATURE_SINGLE_FILE_DIAGNOSTICS)
   set(FEATURE_SINGLE_FILE_DIAGNOSTICS 1)
 endif(NOT DEFINED FEATURE_SINGLE_FILE_DIAGNOSTICS)
+
+if (CLR_CMAKE_TARGET_OSX)
```

Comment:
is this needed? I don't think `FEATURE_COMWRAPPERS` is here. Should it be added?

Reply:
We need this because we condition things in CMake on this feature flag.

Since we only condition things in source for ComWrappers, we didn’t need a flag here. We could add one though and only include comwrappers.cpp and the like in the build when FEATURE_COMWRAPPERS is defined in CMake. Whatever you choose.

Reply:
Ah. I see that now. Nope that makes sense as is. Thanks.